tests/integration_tests/canal_json_storage_partition_table/data/prepare.sql

```diff
@@ -21,10 +21,11 @@ alter table t1 drop partition p1;
 insert into t1 values (7),(8),(9);
 update t1 set a=a+10 where a=9;
 
-create table t2 (a int primary key);
-ALTER TABLE t1 EXCHANGE PARTITION p3 WITH TABLE t2;
-insert into t2 values (100),(101),(102),(103),(104),(105); /*these values will be replicated to in downstream t2*/
-insert into t1 values (25),(29); /*these values will be replicated to in downstream t1.p3*/
+/* TODO(CharlesCheung): EXCHANGE PARTITION will be supported in the future */
```

Comment:
Please remove this commented code and open an issuer for it.

Reply:
https://github.com/pingcap/tiflow/issues/8914